xclim/testing/helpers.py

```diff
@@ -195,3 +199,42 @@ def add_example_file_paths(cache_dir: Path) -> dict[str]:
             ns[f"{variable}_dataset"] = ds.get(variable)
 
     return ns
+
+
+@pytest.fixture
+def test_timeseries():
+    def _test_timeseries(
+        values, variable, start="7/1/2000", units=None, freq="D", as_dataset=False
+    ):
```

Comment:
I suggest `test_timeseries` as a simple function (your `_test_timeseries`) in this module.

Then we could either have a fixture in `conftest`:
```
from xclim.testing.helpers import test_timeseries

@pytest.fixture
def timeseries():
    return test_timeseries
```
(The "test_" prefix has a meaning for pytest, I wouldn't use it in a fixture)

OR

we simply import test_timeseries everywhere.

Reply:
that works! thanks!